docs/sources/datasources/pyroscope/configure-pyroscope-data-source.md

```diff
@@ -33,98 +33,80 @@ refs:
       destination: /docs/grafana/<GRAFANA_VERSION>/datasources/tempo/configure-tempo-data-source/
     - pattern: /docs/grafana-cloud/
       destination: docs/grafana-cloud/connect-externally-hosted/data-sources/tempo/configure-tempo-data-source/
+  provisioning-data-sources:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#data-sources
+    - pattern: /docs/grafana-cloud/provision
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#data-sources
 ---
 
 # Configure the Grafana Pyroscope data source
 
-To configure basic settings for the data source, complete the following steps:
-
-1. Click **Connections** in the left-side menu.
-1. Under Your connections, click **Data sources**.
-1. Enter `Grafana Pyroscope` in the search bar.
-1. Select **Add new data source**.
-1. Click **Grafana Pyroscope** to display the **Settings** tab of the data source.
-1. Set the data source's basic configuration options.
-1. Select **Save & test**.
-
-## Configuration options
+The Pyroscope data source sets how Grafana connects to your Pyroscope database.
 
-You can configure several options for the Pyroscope data source, including the name, HTTP, authentication, querying, and private data source connect.
+You can configure the data source using either the data source interface within Grafana or using a configuration file.
+This page explains how to set up and enable the data source capabilities using Grafana.
 
 If you make any changes, select **Save & test** to preserve those changes.
 
-![Configuration options for the Pyroscope data source](/media/docs/grafana/data-sources/screenshot-pyroscope-data-source-config.png)
-
-### Name and default
+If you are using your own installation of Grafana, you can provision the Pyroscope data source using a YAML configuration file.
+For more information about provisioning and available configuration options, refer to [Provisioning Grafana](ref:provisioning-data-sources).
 
-**Name**
-: Enter a name to specify the data source in panels, queries, and Explore.
+## Before you begin
 
-**Default**
-: The default data source is pre-selected for new panels.
+To configure a Pyroscope data source, you need administrator rights to your Grafana instance and a Pyroscope instance configured to send data to Grafana.
 
-### HTTP
+If you are provisioning a Pyroscope data source, then you also need administrative rights on the server hosting your Grafana instance.
 
-The HTTP section is shown in number 1 in the screenshot.
+## Set up the data source
```

Comment:
Is this procedure lifted from other data sources? It doesn't quite match the current UI. I've tried to fix it.

Reply:
I'll review this with the UI and verify the steps. Thank you for your diligence on this PR.